pkg/agent/openflow/client.go

```diff
@@ -140,12 +141,14 @@ func (c *client) deleteFlows(cache *flowCategoryCache, flowCacheKey string) erro
 	return nil
 }
 
-func (c *client) InstallNodeFlows(hostname string, localGatewayMAC net.HardwareAddr, peerGatewayIP net.IP, peerPodCIDR net.IPNet, tunnelPeerAddr net.IP) error {
-	flows := []binding.Flow{
-		c.arpResponderFlow(peerGatewayIP),
-		c.l3FwdFlowToRemote(localGatewayMAC, peerPodCIDR, tunnelPeerAddr),
+func (c *client) InstallNodeFlows(hostname string, localGatewayMAC net.HardwareAddr, peerGatewayIP net.IP, peerPodCIDR net.IPNet, tunnelPeerAddr net.IP, tunOFPort uint32) error {
+	flows := make([]binding.Flow, 2, 3)
+	flows[0] = c.arpResponderFlow(peerGatewayIP)
+	flows[1] = c.l3FwdFlowToRemote(localGatewayMAC, peerPodCIDR, tunnelPeerAddr, tunOFPort)
+	if peerGatewayIP == nil {
```

Comment:
From the caller of InstallNodeFlows, peerGatewayIP is always not nil. Do you mean check of `tunnelPeerAddr` here?

Reply:
Yes. My mistake. Should be tunnelPeerAddr. Good catch.